theory/python_bindings/_countpairs.c

```diff
@@ -143,11 +143,13 @@ static PyMethodDef module_methods[] = {
      "verbose : boolean (default false)\n"
      "    Boolean flag to control output of informational messages\n\n"
 
-     "boxsize : double\n"
-     "    The side-length of the cube in the cosmological simulation.\n"
+     "boxsize : 3-tuple of double\n"
+     "    The (X,Y,Z) side lengths of the spatial domain.\n"
      "    Present to facilitate exact calculations for periodic wrapping.\n"
-     "    If boxsize is 0., then the wrapping is done based on\n"
-     "    the maximum difference within each dimension of the X/Y/Z arrays.\n\n"
+     "    If the boxsize in a dimension is 0., then\n"
+     "    then that dimension's wrap is done based on the extent of the particle\n"
+     "    distribution. If the boxsize in a dimension is -1., then periodicity\n"
```

Comment:
So this might need to be "If the boxsize in a dimension is *negative*, then ..."